install_homeassistant.sh

```diff
@@ -1,8 +1,19 @@
 #!/bin/sh
```

Comment:
I was thinking about this too - it's a breaking change for first installation, which I'm not happy with.  I'm sorta thinking that having it available isn't a bad idea, as it allows for reinstalls of homeassistant to happen from the hassbian-config script; otherwise users have to find out how to perform this one install themselves.  I did inhibit it from being viewed as a "suite" though, because it shouldn't ordinarily be run...
With a little more effort, it may be possible to detect that install_homeassistant.sh is being run at first-run time, and make it work as it has previously.  It's a case of changing the very last line of the script.
If you're interested, I could look into making this change; but I'm at work and so it won't be a speedily forthcoming one :(